sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-jedis/src/test/java/com/azure/messaging/eventhubs/checkpointstore/jedis/JedisRedisCheckpointStoreTests.java

```diff
@@ -0,0 +1,172 @@
+                assertEquals("consumerGroup", checkpointTest.getConsumerGroup());
+            })
+            .verifyComplete();
+    }
+
+    @Test
+    public void testListCheckpointsEmptyList() {
+        //arrange
+        //act
+        when(jedisPool.getResource()).thenReturn(jedis);
+        when(jedis.smembers("//")).thenThrow(new IllegalArgumentException());
+        //assert
+        Assertions.assertThrows(IllegalArgumentException.class, () -> store.listCheckpoints("", "", ""));
+    }
+
+    @Test
+    public void testCheckpointKeyNotStored() {
+        //arrange
+        Set<String> value = new HashSet<>();
+        value.add("fullyQualifiedNamespace/eventHubNamespace/consumerGroup/one");
+        //act
+        when(jedisPool.getResource()).thenReturn(jedis);
+        when(jedis.smembers("fullyQualifiedNamespace/eventHubName/consumerGroup")).thenReturn(value);
+        when(jedis.hmget(eq("fullyQualifiedNamespace/eventHubNamespace/consumerGroup/one"),
+            eq("checkpoint"))).thenThrow(new NoSuchElementException());
+        //assert
+        Assertions.assertThrows(NoSuchElementException.class, () -> store.listCheckpoints("fullyQualifiedNamespace", "eventHubName", "consumerGroup"));
+    }
+
+    @Test
+    public void testListOwnership() {
+        //arrange
+        PartitionOwnership partitionOwnership = new PartitionOwnership()
+            .setFullyQualifiedNamespace("fullyQualifiedNamespace")
+            .setEventHubName("eventHubName")
+            .setConsumerGroup("consumerGroup")
+            .setPartitionId("one")
+            .setOwnerId("ownerOne")
+            .setETag("eTag");
+
+        Set<String> value = new HashSet<>();
+        List<String> list = new ArrayList<>();
+        value.add("fullyQualifiedNamespace/eventHubNamespace/consumerGroup/one");
+        byte[] bytes = jsonSerializer.serializeToBytes(partitionOwnership);
+        list.add(new String(bytes));
+
+        //act
+        when(jedisPool.getResource()).thenReturn(jedis);
+        when(jedis.smembers("fullyQualifiedNamespace/eventHubName/consumerGroup")).thenReturn(value);
+
+        when(jedis.hmget(eq("fullyQualifiedNamespace/eventHubNamespace/consumerGroup/one"),
+            eq("partitionOwnership"))).thenReturn(list);
+        //assert
+        StepVerifier.create(store.listOwnership("fullyQualifiedNamespace", "eventHubName", "consumerGroup"))
+            .assertNext(partitionOwnershipTest -> {
+                assertEquals("fullyQualifiedNamespace", partitionOwnershipTest.getFullyQualifiedNamespace());
+                assertEquals("eventHubName", partitionOwnershipTest.getEventHubName());
+                assertEquals("consumerGroup", partitionOwnershipTest.getConsumerGroup());
+                assertEquals("ownerOne", partitionOwnershipTest.getOwnerId());
+            })
+            .verifyComplete();
+    }
+
+    @Test
+    public void testListOwnershipEmptyList() {
+        //arrange
+        //act
```

Comment:
I'd remove the act since it is on line 137... and also the same comment applies, since listOwnership returns a Flux, users would expect that the error be propagated through flux's error rather than throwing.